addons/binding/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/handler/NukiSmartLockHandler.java

```diff
@@ -42,43 +42,53 @@ public NukiSmartLockHandler(Thing thing) {
     @Override
     public void initialize() {
         logger.debug("NukiSmartLockHandler:initialize()");
-        String nukiId = (String) this.getConfig().get(NukiBindingConstants.CONFIG_NUKI_ID);
-        BridgeLockStateResponse bridgeLockStateResponse = new NukiHttpClient(getBridgeConfig())
-                .getBridgeLockState(nukiId);
-        if (bridgeLockStateResponse.getStatus() == 200) {
-            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE,
-                    "Nuki Smart Lock is " + bridgeLockStateResponse.getBridgeLockState().getStateName());
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                    bridgeLockStateResponse.getMessage());
+        try {
+            String nukiId = (String) this.getConfig().get(NukiBindingConstants.CONFIG_NUKI_ID);
+            BridgeLockStateResponse bridgeLockStateResponse = new NukiHttpClient(getBridgeConfig())
+                    .getBridgeLockState(nukiId);
+            if (bridgeLockStateResponse.getStatus() == 200) {
+                updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE,
+                        "Nuki Smart Lock is " + bridgeLockStateResponse.getBridgeLockState().getStateName());
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        bridgeLockStateResponse.getMessage());
+            }
+        } catch (NukiBridgeHandlerNotAvailableException e) {
+            logger.error(e.getMessage());
```

Comment:
Can you check for occurrences of this, It's preferred you just assigns a constant string and (optionally) have a placeholder for the real message.